src/rest-api.js

```diff
@@ -83,12 +83,53 @@ const jobName = (labels) => {
 
 const jobLabels = labels => _.omit(labels, (value, key) => isId(key))
 
+const splitLongLabel = (name, value) => {
+  if(value.length > 61) { // 63 (max length)  - 2 (for 'Z')
+    return _.map(
+      _.range(
+        Math.ceil(value.length / 61)
+      ), (idx) => [`${name}${idx}`, `Z${value.substring(61 * idx, 61 * idx + 61)}Z`]
+    )
+  } else
+  return [[name, `Z${value}Z`]]
+}
+const cleanLabel = (label) =>  label.replace(/^Z|Z$/g, '')
+const sanitizeLabels = (labels) => _.object(_.flatten(_.map(_.keys(labels), (key) => splitLongLabel(key, labels[key])), true))
+const cleanLabels = (labels) => _.object(_.map(_.keys(labels), (key) => [key, cleanLabel(labels[key])]))
+
+const combineLongLabels = (labels) => {
+	let processedKeys = []
+	return _.object(_.map(_.keys(labels), (key) => {
```

Comment:
Should you add 'numeric suffix sort' for labels here before the combining operation? To have a predictable order of substrings? or it is done by something else?

Reply:
JS does have a guaranteed property order that was standardised in ES2015.
Before that it was a pseudo-standard among most engines.
https://tc39.es/ecma262/#sec-ordinaryownpropertykeys

I have a test for this behaivour.